output.c

```diff
@@ -112,6 +132,7 @@ output_enable(struct cg_output *output)
 		output_layout_add_auto(output);
 	}
 
+	update_cursor_map_to_output(output->server);
```

Comment:
Are you sure you have to call this function in other case than `handle_new_output()` and `handle_output_destroy()`?

Reply:
Honestly, no I’m not sure. I think that it’s only needed when the outputs change but I only have limited experience with wlroots. 

Happy to make any changes to this which you recommend.